Library/Homebrew/resource.rb

```diff
@@ -198,6 +199,7 @@ def url(val = nil, **specs)
     @url = URL.new(val, specs)
     @downloader = nil
     @download_strategy = @url.download_strategy
+    @url.to_s
```

Comment:
This is technically a breaking change – `url` with an explicit `val` parameter would return the download strategy before, which I'm hoping was not used. That would result in a clumsy return type, requiring `T.cast`s elsewhere.